src/main.ts

```diff
@@ -1,230 +1,21 @@
-import {
-  path,
-  serveDir,
-  serveFile,
-  Server,
-  Status,
-  STATUS_TEXT,
-} from "./deps.ts";
+import frontend from "ghact/src/frontend.tsx"
```

Comment:
Why is this not imported via depts.ts? This will make builds take longer and have worse caching for no apparent reason